app/scripts/views/complete_sign_up.js

```diff
@@ -44,6 +45,10 @@ function (Cocktail, FormView, BaseView, CompleteSignUpTemplate,
       // cache the email in case we need to attempt to resend the
       // verification link
       this._email = this._account.get('email');
+      this._able = options.able;
+
+      this._experiments = new ExperimentInterface();
+      this._experiments.chooseExperiments();
```

Comment:
Totally mistyped the name from memory. But yeah, views could assume `this.experiments` (no underscore since it'd be public). Or if there are only a few methods that views use on `experiments` the mixin could define them directly and delegate to `_experiments`, keeping it private.